[SecuritySolution] Numbers of Network KPI metric are cropped

agusruidiazgd opened the pull request:
## Summary

Fix issue https://github.com/elastic/kibana/issues/210806


### Network Page
| Before | After |
|--------|-------|
| ![Network Before](https://github.com/user-attachments/assets/0b3277f7-dbe2-41a4-ac76-05a806bb42e8) | ![Network After](https://github.com/user-attachments/assets/12b16e81-d13b-495f-9d46-a4154aa8e0cf) |

### Host Page
| Before | After |
|--------|-------|
| ![Host Before](https://github.com/user-attachments/assets/1338b0e5-f4b1-4d1a-ad54-da392f0eff07) | ![Host After](https://github.com/user-attachments/assets/48af01d9-a4aa-450d-ac81-e2b4a724363f) |

### Users Page
| Before | After |
|--------|-------|
| ![Users Before](https://github.com/user-attachments/assets/d7acdf4d-9a4f-4b67-923c-a184e13b367f) | ![Users After](https://github.com/user-attachments/assets/43bf36bd-1e3b-47d7-b121-0d4142b18cbe) |


### Checklist

Check the PR satisfies following conditions. 

Reviewers should verify this PR satisfies this list as well.

- [x] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios




<!--ONMERGE {"backportTargets":["8.18","8.x","9.0"]} ONMERGE-->

agusruidiazgd committed:
increase min-width

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

agusruidiazgd committed:
fix min width depending if icon exists

agusruidiazgd commented on x-pack/solutions/security/plugins/security_solution/public/explore/components/stat_items/metric_embeddable.tsx:
```diff
@@ -27,44 +28,53 @@ const MetricEmbeddableComponent = ({
-                    timerange={timerange}
-                    inspectTitle={inspectTitle}
+      {fields.map((field) => {
+        const metricItemStyles =
+          !field.icon &&
+          css`
+            &.euiFlexItem {
+              min-width: 250px;
+            }
+          `;
```
If the field does not have an icon to render, increase the `min-width` to prevent the number from being cropped.

agusruidiazgd commented:
@elasticmachine merge upstream

elasticmachine committed:
Merge branch 'main' into fix/network-cropped-metrics-issue-210806

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/276977)
* Commit: 0a4e29fe28551565e08343ceede92e83f238b5de

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/0dfa21571b7e8d2a9c6fe55d899750ba2e23e5f9) | [after](https://github.com/elastic/kibana/commit/0a4e29fe28551565e08343ceede92e83f238b5de) | diff |
| --- | --- | --- | --- |
| `securitySolution` | 8.9MB | 8.9MB | +98.0B |



cc @agusruidiazgd

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/276977","number":276977,"commit":"0a4e29fe28551565e08343ceede92e83f238b5de"}],"number":276977}
buildkite-pr-comment-->

kapral18 committed:
fix: simplify MetricEmbeddableComponent and change metric alignment to center

- Change `autoScaleMetricAlignment` from "left" to "center" in various network KPI files
- Simplify `MetricEmbeddableComponent` and remove unneeded styles
- Remove `MetricItem` styled component and associated CSS

agusruidiazgd committed:
Merge branch 'main' into fix/network-cropped-metrics-issue-210806

agusruidiazgd committed:
Merge branch 'main' into fix/network-cropped-metrics-issue-210806

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/277646)
* Commit: 9cca9a84378fa43c0fd071b5731bf119eefc410f

### Failed CI Steps
* [Defend Workflows Cypress Tests #6](https://buildkite.com/elastic/kibana-pull-request/builds/277646#01951e23-8150-4737-86d3-9430b65f659a)

### History
* :green_heart: [Build #276977](https://buildkite.com/elastic/kibana-pull-request/builds/276977) succeeded 0a4e29fe28551565e08343ceede92e83f238b5de

cc @agusruidiazgd

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/277646","number":277646,"commit":"9cca9a84378fa43c0fd071b5731bf119eefc410f"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/276977","number":276977,"commit":"0a4e29fe28551565e08343ceede92e83f238b5de"}],"number":277646}
buildkite-pr-comment-->

agusruidiazgd committed:
Merge branch 'main' into fix/network-cropped-metrics-issue-210806

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/277708)
* Commit: 2642a580ce0d122ff9ce679c23393b946008e6a7

### Failed CI Steps
* [FTR Configs #39](https://buildkite.com/elastic/kibana-pull-request/builds/277708#01951ec1-7010-41c3-9819-ea5073dc48fa)
* [FTR Configs #110](https://buildkite.com/elastic/kibana-pull-request/builds/277708#01951ec1-704a-4102-bc49-cf99152324d0)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/277708#01951ec1-704a-4102-bc49-cf99152324d0) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/277708/jobs/01951ec1-704a-4102-bc49-cf99152324d0/artifacts/01951ee0-4e75-4d02-9ead-9c85abd4dc14) FTR Configs #<span></span>110 / Fleet Endpoints fleet_setup upgrade managed package policies should upgrade managed package policies
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/277708#01951ec1-7010-41c3-9819-ea5073dc48fa) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/277708/jobs/01951ec1-7010-41c3-9819-ea5073dc48fa/artifacts/01951ee8-6dde-43bd-a4c7-735afd2bb244) FTR Configs #<span></span>39 / Space awareness change space agent policies PUT /agent_policies/{id} should allow set policy in multiple space

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/b85919e6c299e67121e47b542762dfe124b99689) | [after](https://github.com/elastic/kibana/commit/2642a580ce0d122ff9ce679c23393b946008e6a7) | diff |
| --- | --- | --- | --- |
| `securitySolution` | 8.9MB | 8.9MB | -150.0B |



### History
* :broken_heart: [Build #277646](https://buildkite.com/elastic/kibana-pull-request/builds/277646) failed 9cca9a84378fa43c0fd071b5731bf119eefc410f
* :green_heart: [Build #276977](https://buildkite.com/elastic/kibana-pull-request/builds/276977) succeeded 0a4e29fe28551565e08343ceede92e83f238b5de

cc @agusruidiazgd

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/277708","number":277708,"commit":"2642a580ce0d122ff9ce679c23393b946008e6a7"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/277646","number":277646,"commit":"9cca9a84378fa43c0fd071b5731bf119eefc410f"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/276977","number":276977,"commit":"0a4e29fe28551565e08343ceede92e83f238b5de"}],"number":277708}
buildkite-pr-comment-->

elasticmachine committed:
Merge branch 'main' into fix/network-cropped-metrics-issue-210806

kibanamachine commented:
Starting backport for target branches: 8.18, 8.x, 9.0

https://github.com/elastic/kibana/actions/runs/13500153477

kibanamachine commented:
## 💔 Some backports could not be created

| Status | Branch | Result |
|:------:|:------:|:------|
|❌|8.18|Backport failed because of merge conflicts|
|✅|8.x|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/212249">](https://github.com/elastic/kibana/pull/212249)|
|❌|9.0|Backport failed because of merge conflicts|

Note: Successful backport PRs will be merged automatically after passing CI.

### Manual backport
To create the backport manually run:
```
node scripts/backport --pr 211460
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 9.6.6 -->
